lib/Dialect/TritonGPU/Transforms/Utility.cpp

```diff
@@ -671,131 +671,180 @@ auto getMfmaInsnGroupAttrMap = []() -> const MfmaInsnGroupMap & {
   static MfmaInsnGroupMap MfmaInsnMap{
       // f32
       // mfma_f32_32x32x2f32
-      {{32, MfmaTypeId::Fp32TyId, 1},
+      {{32, 32, MfmaTypeId::Fp32TyId, 1},
        {32, 32, 2, 1, ROCDL::mfma_f32_32x32x2f32::getOperationName()}},
-      {{32, MfmaTypeId::Fp32TyId, 2},
+      {{32, 32, MfmaTypeId::Fp32TyId, 2},
        {32, 32, 2, 1, ROCDL::mfma_f32_32x32x2f32::getOperationName()}},
-      {{32, MfmaTypeId::Fp32TyId, 3},
+      {{32, 32, MfmaTypeId::Fp32TyId, 3},
        {32, 32, 2, 1, ROCDL::mfma_f32_32x32x2f32::getOperationName()}},
       // mfma_f32_16x16x4f32
-      {{16, MfmaTypeId::Fp32TyId, 1},
+      {{16, 16, MfmaTypeId::Fp32TyId, 1},
        {16, 16, 4, 1, ROCDL::mfma_f32_16x16x4f32::getOperationName()}},
-      {{16, MfmaTypeId::Fp32TyId, 2},
+      {{16, 16, MfmaTypeId::Fp32TyId, 2},
        {16, 16, 4, 1, ROCDL::mfma_f32_16x16x4f32::getOperationName()}},
-      {{16, MfmaTypeId::Fp32TyId, 3},
+      {{16, 16, MfmaTypeId::Fp32TyId, 3},
        {16, 16, 4, 1, ROCDL::mfma_f32_16x16x4f32::getOperationName()}},
       // mfma_f32_4x4x1f32
-      {{4, MfmaTypeId::Fp32TyId, 1},
+      {{4, 4, MfmaTypeId::Fp32TyId, 1},
        {4, 4, 16, 1, ROCDL::mfma_f32_4x4x1f32::getOperationName()}},
```

Comment:
I think I made a mistake here. 
It should be 4 , 4, 1, 1 instead of 4, 4, 16, 1

Reply:
Oh, I realized this is for (4x16) x (16x4) --> 4x4. nvm